pkg/controller/podautoscaler/horizontal.go

```diff
@@ -229,13 +229,28 @@ func (a *HorizontalController) processNextWorkItem() bool {
 	return true
 }
 
+// Types to help keep status and spec separate and avoid a positive
+// feedback loop.  Generally, nothing from status should go back into
+// spec.
+type specReplicas int32
```

Comment:
Where does specReplicas go? To Deployment/RS specification (scale subresource)?

Reply:
specReplicas goes to the scale subresource (spec).

Re: statusReplicas, I removed it from all the places it wasn't needed.  I tried a few different approaches to fixing this problem and that plumbing was just a leftover.